thermo/data/co2-thermo.yaml

```diff
@@ -0,0 +1,62 @@
+description: |-
+  CO2 phases based on the equation of state 
+
+cantera-version: 2.5.0a3
+date: Wed, 10 Jul 2020 11:38:00 +0000
+
+units: {length: cm, time: s, quantity: mol, activation-energy: cal/mol}
+
+phases:
+- name: CO2-Ideal
+  thermo: ideal-gas
+  elements: [O, C]
+  species: [CO2]
+  kinetics: gas
+  transport: mixture-averaged
+  state: {T: 300.0, P: 1 atm}
+
+- name: CO2-RK
+  thermo: Redlich-Kwong
+  elements: [O, C]
+  species: [{rk-species: [CO2]}]
+  kinetics: gas
+  transport: mixture-averaged
```

Comment:
Since transport is not used in this example, you can delete this specification, and the transport data for the ideal gas species.

Reply:
Transport properties are deleted now.